Automattic/wp-calypso
# Use material icons for the Me page

#### Changes proposed in this Pull Request

* Replaces gridicons with Material icons in the Me section
* Introduces logic in the [sidebar item component](https://github.com/Automattic/wp-calypso/blob/master/client/layout/sidebar/item.jsx) to render either the Gridicon or the Material icon, not both.

Before:
<img width="230" alt="Screen Shot 2019-05-06 at 4 22 18 PM" src="https://user-images.githubusercontent.com/5375500/57252826-30ffa900-701b-11e9-931c-44a398aa818c.png">

After:
<img width="239" alt="Screen Shot 2019-05-06 at 4 22 25 PM" src="https://user-images.githubusercontent.com/5375500/57252835-352bc680-701b-11e9-8296-54ad4481d217.png">

#### Testing instructions

* Visit wordpress.com/me when logged in
* Verify that the proper material icons are in use 

Fixes #31770
